.github/workflows/release.yml

```
@@ -0,0 +1,58 @@
+name: aiodynamo release workflow
+
+on:
+  release:
+    types: [released] # only triggered on release, not pre-release or draft release
+
+permissions:
+  contents: read
+  id-token: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  PYTHON_VERSION: 3.11
+  TARGET_BRANCH: ${{ github.event.release.target_commitish }}
+  DEFAULT_BRANCH: ${{ github.event.repository.default_branch }}
+  TAG_NAME: ${{ github.ref_name }}
+
+jobs:
+  release:
+    name: Release to PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check whether release target branch is default branch
+        if: env.TARGET_BRANCH != env.DEFAULT_BRANCH
+        uses: actions/github-script@v6
+        with:
+          script: |
+            core.setFailed('Target branch (${{ env.TARGET_BRANCH }}) is not default branch (${{ env.DEFAULT_BRANCH }}).')
+      
+      - uses: actions/checkout@v3
+      
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PYTHON_VERSION_MIN }}
```

You're renamed the env var declaration...
```suggestion
          python-version: ${{ env.PYTHON_VERSION }}
```